File: app/src/main/java/com/example/billeteramodulo4/Home_Page_Empty_Case.java
package com.example.billeteramodulo4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;


public class Home_Page_Empty_Case extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.home_page_empty_case);

        // Abrir nueva activity

        ImageButton boton = findViewById(R.id.sendhome);

        boton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                //ir a activity 5
                Intent i = new Intent(Home_Page_Empty_Case.this, Request_Money.class);
                startActivity(i);
            }
        });


    }
}

File: app/src/main/java/com/example/billeteramodulo4/HomePageEmptyCase.kt
package com.example.billeteramodulo4

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import com.example.billeteramodulo4.databinding.FragmentHomePageEmptyCaseBinding

class HomePageEmptyCase : Fragment() {

    private lateinit var binding: FragmentHomePageEmptyCaseBinding


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {

        binding = FragmentHomePageEmptyCaseBinding.inflate(inflater, container, false)
        val root = binding.root
        // Inflate the layout for this fragment

        binding.enviardineroemptypage.setOnClickListener {
            findNavController().navigate(R.id.action_homePageEmptyCase_to_sendMoney)
        }
        binding.ingresardineroemptypage.setOnClickListener {
            findNavController().navigate(R.id.action_homePageEmptyCase_to_requestMoney)
        }

        binding.profileuserempty.setOnClickListener {
            findNavController().navigate(R.id.action_homePageEmptyCase_to_profilePage)
        }
        return root
    }


}